Source/Core/Core/Core.vcxproj

```diff
@@ -35,6 +35,11 @@
     <Import Project="..\..\VSProps\PCHUse.props" />
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
+    <ItemDefinitionGroup>
```

Comment:
As far as I'm aware dependencies go in Base.props. (@shuffle2 correct me if I'm wrong with the VS stuff)

Reply:
Seems like DolphinWX.vcxproj includes dependencies this way? Base.props doesn't have any included dependencies.

Reply:
Lib dependencies are normally auto-figured out by msbuild dependency tree. Where even is qwave.lib?

Reply:
If it can't be done "properly" then put it in the vcxproj of the final binaries which need to link against the lib (which is why DolphinWX.vcxproj has a bunch).

Reply:
https://msdn.microsoft.com/en-us/library/windows/desktop/aa373381(v=vs.85).aspx
I believe it's just a part of the Windows SDK?